packages/compiler/core/parser.ts

```diff
@@ -44,6 +44,7 @@ import {
   Node,
   NodeFlags,
   NumericLiteralNode,
+  OperationSignature,
```

Comment:
Remember to update the grammar in spec.emu.html

Reply:
Done!  Let me know whether it looks wrong and I can tweak it.  Do I need to do anything else to update the tmlanguage grammar?